src/Command/Model/FileTemplate.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Centreon\Command\Model;
+
+abstract class FileTemplate
+{
+    protected string $licenceHeader;
+
+    public function __construct()
+    {
+        $date = date("Y");
+        $this->licenceHeader =
```

Comment:
We should think to reuse the **`PhpCsFixerRuleSet::getLicenseHeader()`** which is currently private.
This may avoid problems between the generation and the current php-cs-fixer rules.

Reply:
It doesn't seem to work with `PhpCsFixerRuleSet::getLicenseHeader()` method. It outputs a header as a string without comment symbols (`/*...`)

Reply:
1️⃣ Change `PhpCsFixerRuleSet::detectCentreonProjectLicense()` to be `public`
2️⃣ Add the following method to `PhpCsFixerRuleSet` class
```php
    /**
     * Get the license header as a php comment.
     *
     * @param string $projectLicense
     *
     * @return string
     */
    public static function getLicenseHeaderAsPhpComment(string $projectLicense): string
    {
        if ('' === ($header = self::getLicenseHeader($projectLicense))) {
            return '';
        }

        $lines = explode("\n", $header);
        $lines = array_map(static fn(string $line): string => rtrim(' * ' . $line), $lines);

        return "/*\n" . implode("\n", $lines) . "\n */\n";
    }
```
3️⃣ Change the `FileTemplate` construct :
```php
    public function __construct()
    {
        $licence = PhpCsFixerRuleSet::detectCentreonProjectLicense(__DIR__);
        if ('' !== $licence) {
            $this->licenceHeader = PhpCsFixerRuleSet::getLicenseHeaderAsPhpComment($licence);
        } else {
            $this->licenceHeader = '';
        }
    }
```